eslint-config-internal/package.json

```diff
@@ -12,12 +12,19 @@
     "type": "git",
     "url": "git://github.com:mate-academy/fed.git"
   },
+  "_moduleAliases": {
+    "@": "./src"
+  },
```

Comment:
we probably don't need it in the global config
src here may point to different folder